docs/manual/getting_started.rst

```diff
@@ -581,51 +686,51 @@ Data-Centric Architecture
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In a service-centric architecture, to interact, applications need to
-know each other's interfaces to share data, share events, share
-commands, or replies. These interfaces are modeled as sets of operations
+know each other's interfaces to share data, share events, and share
+commands or replies. These interfaces are modeled as sets of operations
 and functions that are managed in centralized repositories. This kind of
-architecture creates unnecessary dependencies that end-up creating a
+architecture creates unnecessary dependencies that create a
 tightly coupled system. The centralized interface repositories are
 usually seen as a single point of failure.
 
 In a data-centric architecture, your design focuses on the data each
```

Comment:
Have we decided on American or UK spelling for these documents? If it's UK spelling this should be:
```suggestion
In a data-centric architecture, your design focusses on the data each
```
The decision should be added to a style guide.